packages/scripts/bin/wp-scripts.js

```diff
@@ -3,8 +3,12 @@
 /**
  * Internal dependencies
  */
-const { getArgsFromCLI, spawnScript } = require( '../utils' );
+const { getNodeArgsFromCLI, spawnScript } = require( '../utils' );
 
-const [ scriptName, ...nodesArgs ] = getArgsFromCLI();
+const { scriptName, scriptArgs, nodeArgs } = getNodeArgsFromCLI( [
+	'--inspect-brk',
```

Comment:
I did some research and it looks like node take can a large number of CLI options:

https://nodejs.org/api/cli.html

Do you know whether other options would be useful here?

What other approaches did you consider?

One thing I discovered is `--` option:
> Indicate the end of node options. Pass the rest of the arguments to the script. If no script filename or eval/print script is supplied prior to this, then the next argument will be used as a script filename.

It looks like a perfect fit here but I don't know how it would play out in this specific case.

We already use `minimist` package for parsing args and it has support for this option:
https://www.npmjs.com/package/minimist#var-argv--parseargsargs-opts

Reply:
> Still, considering how similar `wp-scripts` is to `react-scripts`, I think it would be best to follow the same pattern they use. Their way of passing args seems fine from the user's perspective, and if some people are already familiar with how it works there, it doesn't make sense to me to do something different that might cause confusion unless the new way was going to fulfill a unique use-case for us.

Thanks for reminder about `react-scripts`, it’s the first place I should check. This is how they have implemented it:

https://github.com/facebook/create-react-app/blob/e89f153224cabd67efb0175103244e0b7f702767/packages/react-scripts/bin/react-scripts.js#L21L25 

They have a very simple approach that addresses my main concern which is the list of node args that needs to be maintained on our side. I wouldn’t mind if we replicate this implementation. They have only 4 script names there so they hardcode them, but we can get all names from the `scripts` subfolder based on the file located there.